File: src/APIBundle/Entity/ComcheckStatus.php
<?php

namespace APIBundle\Entity;

use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity()
 * @ORM\Table(name="comchecks_statuses")
 */
class ComcheckStatus
{
    /**
     * @ORM\Id
     * @ORM\Column(type="string")
     * @ORM\GeneratedValue(strategy="UUID")
     */
    protected $status_id;

    /**
     * @ORM\ManyToOne(targetEntity="User", inversedBy="user_comchecks_statuses")
     * @ORM\JoinColumn(name="status_user", referencedColumnName="user_id", onDelete="CASCADE")
     * @var USer
     */
    protected $status_user;

    /**
     * @ORM\ManyToOne(targetEntity="Comcheck", inversedBy="comcheck_comchecks_statuses")
     * @ORM\JoinColumn(name="status_comcheck", referencedColumnName="comcheck_id", onDelete="CASCADE")
     * @var Comcheck
     */
    protected $status_comcheck;

    /**
     * @ORM\Column(type="datetime")
     */
    protected $status_last_update;

    public function getStatusId()
    {
        return $this->status_id;
    }

    public function setStatusId($id)
    {
        $this->status_id = $id;
        return $this;
    }

    public function getStatusUser()
    {
        return $this->status_user;
    }

    public function setStatusUser($user)
    {
        $this->status_user = $user;
        return $this;
    }

    public function getStatusComcheck()
    {
        return $this->status_comcheck;
    }

    public function setStatusComcheck($comcheck)
    {
        $this->status_comcheck = $comcheck;
        return $this;
    }

    public function getStatusLastUpdate()
    {
        return $this->status_last_update;
    }

    public function setStatusLastUpdate($lastUpdate)
    {
        $this->status_last_update = $lastUpdate;
        return $this;
    }
}

File: openex-player/src/main/java/io/openex/database/model/ComcheckStatus.java
package io.openex.database.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import io.openex.helper.MonoModelDeserializer;
import org.hibernate.annotations.GenericGenerator;

import javax.persistence.*;
import java.util.Date;

@Entity
@Table(name = "comchecks_statuses")
public class ComcheckStatus implements Base {
    @Id
    @Column(name = "status_id")
    @GeneratedValue(generator = "UUID")
    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
    @JsonProperty("status_id")
    private String id;

    @ManyToOne
    @JoinColumn(name = "status_user")
    @JsonSerialize(using = MonoModelDeserializer.class)
    @JsonProperty("status_user")
    private User user;

    @ManyToOne
    @JoinColumn(name = "status_comcheck")
    @JsonSerialize(using = MonoModelDeserializer.class)
    @JsonProperty("status_comcheck")
    private Comcheck comcheck;

    @Column(name = "status_last_update")
    @JsonProperty("status_last_update")
    private Date lastUpdate;

    @Column(name = "status_state")
    @JsonProperty("status_state")
    private boolean state;

    @Override
    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }

    public Comcheck getComcheck() {
        return comcheck;
    }

    public void setComcheck(Comcheck comcheck) {
        this.comcheck = comcheck;
    }

    public boolean isState() {
        return state;
    }

    public void setState(boolean state) {
        this.state = state;
    }

    public Date getLastUpdate() {
        return lastUpdate;
    }

    public void setLastUpdate(Date lastUpdate) {
        this.lastUpdate = lastUpdate;
    }
}
